### PR TITLE
Update rails_basics.md

### PR DESCRIPTION
**Pull request by cloudtemplar**
this is the best explanation of REST I've ever seen, I wish everyone who is learning this stuff could watch it

### CODE REVIEW

**Comment by KevinMulhern** [comment]
Great find @cloudtemplar, exposing the student to postman is a great thing to do to. I would argue this resource is too good for the extra resources in this lesson 😄 

What would you think about putting it in [this lesson](https://www.theodinproject.com/courses/ruby-on-rails/lessons/routing) as the second assignment?

**Comment by cloudtemplar** [comment]
Yeah, it fits perfectly in place that you suggested. I didn't really thought it through :)

**Comment by KevinMulhern** [comment]
Hi @cloudtemplar, I see you've added a new project in this PR. That would be better in a separate PR I think 😄 

 I'm not sure if that tutorial fits in with the rails course, something more geared towards rails would be better.

**Comment by cloudtemplar** [comment]
I'm really sorry, I got little experience in contributing. I'll make it
right.

02.07.2017 19:22 "Kevin Mulhern" <notifications@github.com> napisał(a):

Hi @cloudtemplar <https://github.com/cloudtemplar>, I see you've added a
new project in this PR. That would be better in a separate PR I think 😄

I'm not sure if that tutorial fits in with the rails course, something more
geared towards rails would be better.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/TheOdinProject/curriculum/pull/6777#issuecomment-312504851>,
or mute the thread

**Comment by cloudtemplar** [comment]
I think that it's all good now

File: rails/project_apis.md [hunk 1]
```diff
@@ -126,8 +126,7 @@ This project may seem simple, but now you've got a website that is both a normal
 * [Sophia Wu's solution](https://github.com/SophiaLWu/kittens-api)
 * [Daniel Aguilar's solution](https://github.com/danaguilar/odin-kittens)
 
-
-### Project 2: Using a Third Party API
+## Project 2: Using a Third Party API
```
Comment (KevinMulhern) [inline rails/project_apis.md]: all the headings within a lesson need to use `###`, these are h3's in HTML. They styling on the website will mess up if they are different.

File: rails/routing.md [hunk 2]
```diff
@@ -187,7 +187,10 @@ You should have a good sense of what's going on in the routes file by now but pr
 
 1. Read the [Rails Guides chapter on Routing](http://guides.rubyonrails.org/routing.html), sections 1-2.5, 3.1-3.4, and 4.6.
 
-### Additional Resources
+2. Watch this [Wonderful explanation of how REST and HTTP works](https://www.youtube.com/watch?v=Q-BpqyOT3a8)
+Make sure to follow the tutorial!
+
+## Additional Resources
```
Comment (KevinMulhern) [inline rails/routing.md]: dito for this one

**Comment by KevinMulhern** [comment]
Thanks @cloudtemplar 
LGTM 👍

**Pull request merged**